### ISSUE TITLE
Issue when running containers through docker-compose up

### ISSUE THREAD

**Issue by aroop-biswal-vts** [IQTLabs/SkyScan#64]
We are attempting to run this project on this setup:
Software: Latest commit on main branch as of today.
OS - arm64 (2021-10-30 Bullseye)
Hardware - RaspberryPi4

When running docker-compose up, the images are successfully pulled. When running the images, however, we get some errors such as:

" ERROR:root:Unable to detect Coral boardFailed to load delegate from libedgetpu.so.1"

"cp: can't stat '/etc/letsencrypt/live/broker.mqtt.local/*.pem': No such file or directory
mqtt_1          | cat: can't open '/run/secrets/mqtt_user': No such file or directory
mqtt_1          | cat: can't open '/run/secrets/mqtt_pass': No such file or directory"

"axis-ptz_1      |     raise ValueError('Invalid host.')
axis-ptz_1      | ValueError: Invalid host.
"

These errors raised a few questions:
1. Do we need to have a coral board?
2. Is a GPS device required for this project?
3. If a GPS device is required, is there a way to run the project by disabling the need for the GPS?

We are a small group working on our summer internship research project and would greatly appreciate any guidance with getting this project set up. We would appreciate a point of contact and are open to receiving any communications through my email: aroop.biswal@vts-i.com.

Thank you for your time!

**Comment by mchadwick-iqt** [comment]
Hi @aroop-biswal-vts - we're making some updates today to get this to a state where it will just come online.  @meadej will be addressing.  To answer your questions:

1. You don't need a coral board, we accidentally merged some dev work into the main branch.  This will be addressed today
2. We are deprecating GPS support - we did not find it to be accurate enough to reliably get aircraft in frame when zoomed in tight.  We generally use a surveyed position from google maps and put the lat/long into the .env file
3. You can delete the EGI container from the docker-compose file.

**Comment by meadej** [comment]
Hi @aroop-biswal-vts - thank you for bringing up these issues and for your patience. 

As @mchadwick-iqt mentioned above, we have placed some current development work under the main branch. For immediate use in the field, we have set-up a `stable` branch which will work directly out of the box and have cut a release, `SkyScan Version 2.1`, which uses proven-stable containers tested in our lab. 

To use this branch, reference the `docker-compose.yml` file included [here](https://github.com/IQTLabs/SkyScan/blob/stable/docker-compose.yml) and ensure that you have environment variables configured as shown in the example [here](https://github.com/IQTLabs/SkyScan/blob/stable/env-example), paying particular attention to the `RELEASE_NAME` variable. 

Let us know if you have any further questions - they are greatly helpful in our work to build out documentation and improve stability.

Thank you!
